sys-apps/ignition/files/0001-sed-s-coreos-flatcar.patch

```diff
@@ -1,13 +1,13 @@
-From 4b86d64fe196d9cb5ca43548234d521907a1cb4d Mon Sep 17 00:00:00 2001
```

Comment:
Do we still need to carry this patch? I'm not sure what is the point of this rename, especially that we are not using our ignition fork any more.

Reply:
The old fork is still used for the translation